meson.build

```diff
@@ -237,6 +237,12 @@ if r.returncode() == 1 and get_option('subprojects_check')
   error('Subprojects are not updated. Please run `git clean -dxff subprojects/` to delete all local subprojects directories. If you want to compile against current subprojects then set option `subprojects_check=false`.')
 endif
 
+# handle libswift library
```

Comment:
put this just above the libswift subproject call please

Reply:
This was not resolved.

Reply:
at which line did you want this?

Reply:
Just before the libswift_proj, as done for all other subprojects. That means at https://github.com/rizinorg/rizin/pull/1595/files#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4R257 .